assets/i18n/en_US.json

```diff
@@ -47,15 +48,16 @@
     "patcherView": {
         "widgetTitle": "Patcher",
         "patchButton": "Patch",
-        "patchDialogText": "You have selected a resource patch and a split APK installation has been detected, so patching errors may occur.\nAre you sure you want to proceed?"
+        "patchDialogText": "You have selected a resource patch and a split APK installation has been detected, so patching errors may occur.\nAre you sure you want to proceed?",
+        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported. Proceed anyways?"
     },
     "appSelectorCard": {
         "widgetTitle": "Select an application",
         "widgetTitleSelected": "Selected application",
         "widgetSubtitle": "No application selected",
         "noAppsLabel": "No applications found",
         "currentVersion": "Current",
-        "recommendedVersion": "Recommended",
+        "recommendedVersion": "Suggested",
```

Comment:
Everything related to `recommendedVersion` should be `suggestedVersion` instead

Reply:
u fix :trollface: